### ISSUE TITLE
dap-enable-mouse-support causes cursor to jump randomly in Emacs 27.1

### ISSUE THREAD

**Issue by simonpintarelli** [emacs-lsp/dap-mode#338]
There is some weird behavior when scrolling (e.g. press and hold `j` in evil mode) or when navigating through helm-swoop using `C-n`. When scrolling and the same file is opened for example in two buffers side-by-side (within the same frame), the cursor starts jumping randomly between buffers. Similar effects appear in helm-swoop. If Emacs is run in the terminal, it does not occur. It seems the keyboard repeat rate needs to be high enough to trigger the bug.

I couldn't reproduce the problem with Emacs `26.3`, only in `27.1` and the latest emacs-git version.

I'm using latest melpa dap version `20200813.1755`, but haven't tried out previous version, since the upgrade to Emacs 26.3 -> 27.1 happened yesterday.

A temporary workaround is to set  `dap-enable-mouse-support` to nil. Thanks to @yyoncho for the hint!

**Comment by wztdream** [comment]
This should not related with emacs version, I use emacs26.3, have same issue.

**Comment by ericdallo** [comment]
Yeah, I'm this same problem and I had no idea what was messing with my cursor, I don't know why dap-mouse is doing that, any clues how to fix that @yyoncho ?

**Comment by yyoncho** [comment]
I think that it is related to dap-tooltip-activate-mouse-motions
